src/units/pathfinder2remaster/player-core/ancestry/ancestry-template

```diff
@@ -0,0 +1,53 @@
+unit: ancestry/remaster/<ancestry>
+in: ancestry
+group: "_{Player Core}"
+name: "_{<ancestry>}"
```

Comment:
I can see the value in having these templates here.
To ensure they never get included by accident, I'd suggest adding this line to the top section:
```
enabled: false
```
this will prevent them being processed, and can be removed when you copy to a real unit.